tracee-rules/engine/engine.go

```diff
@@ -69,7 +69,6 @@ func NewEngine(sigs []types.Signature, sources EventSources, output chan types.F
 func (engine *Engine) Start(done chan bool) {
 	go engine.consumeSources(done)
 	for s, c := range engine.signatures {
-		defer close(c)
```

Comment:
Let's do that in a dedicated PR after we reach the consensus whether to close or not to close the channel. This change deserves its own commit with a bit of explanation and I don't think we want to smuggle it with benchmarks. What's more we have #777 to track it.

For sure we have to do sth about it as I'm also seeing this warning in my IDEA: **Possible resource leak, 'defer' is called in the 'for' loop**

Reply:
Yeah sorry I didn't mean to include this change in this PR. I'll revert it back.